snuba/datasets/transactions.py

```diff
@@ -175,6 +175,8 @@ def column_expr(self, column_name, query: Query, parsing_context: ParsingContext
             return 'IPv4NumToString(ip_address_v4)'
         if column_name == 'ip_address_v6':
             return 'IPv6NumToString(ip_address_v6)'
+        if column_name == 'event_id':
+            return 'replaceAll(toString(event_id), \'-\', \'\')'
```

Comment:
Does @markstory  know about this change ?

Reply:
@markstory I noticed this didn't work when I was trying to add a test for getting a next/previous event here -> https://github.com/getsentry/sentry/pull/15104. It also avoids exposing 2 different event ID formats to end users of Discover.

Reply:
This will require some coordination with sentry as there is code in the conditions transforms that reformats 32byte hex strings to be 36byte uuid strings with dashes.

I am happy to see that code becoming obselete though as having multiple eventid formats was likely to cause us future pain.

Reply:
As per discussion on #15128 seems this is fine.